Tests/SwiftParserTest/DeclarationTests.swift

```diff
@@ -3334,4 +3334,20 @@ final class DeclarationTests: ParserTestCase {
       )
     )
   }
+
+  func testCoroutineAccessors() {
+    assertParse(
+      """
+      var i_rm: Int {
+        read {
+          yield _i
+        }
+        modify {
+          yield &_i
+        }
+      }
+      """,
+      experimentalFeatures: .coroutineAccessors
+    )
+  }
```

Comment:
Could you also add a test case that `read` and `modify` are not parsed when the `.coroutineAccessors` experimental features is not specified?

Reply:
Yep--added one for each.